src/frontends/common/include/openvino/frontend/visibility.hpp

```diff
@@ -4,21 +4,7 @@
 
 #pragma once
 
-#include "openvino/core/visibility.hpp"
+#include "openvino/core/ov_visibility.hpp"
```

Comment:
why have you renamed `openvino/core/visibility.hpp`?

Reply:
I don't know. I also think that it is not needed.

Reply:
I didn't rename `openvino/core/visibility.hpp` to `openvino/core/ov_visibility.hpp`.
I renamed `openvino/core/core_visibility.hpp` to `openvino/core/ov_visibility.hpp`.

Here I changed include from `visibility.hpp` to `ov_visibility.hpp` because `ov_visibility.hpp` contains `OPENVINO_API` macro.

But in general, I will merge `ov_visibility` and `visibilty` headers